tdp/core/deployment/test_deployment_runner.py

```diff
@@ -111,8 +111,10 @@ def test_deployment_plan_with_filter_is_success(
     deployment_iterator = mock_deployment_runner.run(deployment)
 
     for i, _ in enumerate(deployment_iterator):
```

Comment:
Syntaxe like `for i, (op, fn) in enumerate(deployment_iterator):` does not work ?

Reply:
Sorry didn't see this comment before. yes it works, it has been rewritten